client/components/UI/Form/SelectTagInput/SelectTagPopup.jsx

```diff
@@ -22,17 +23,17 @@ export class SelectTagPopup extends React.Component {
         if (event) {
             switch (event.keyCode) {
             case KEYCODES.ENTER:
-                event.preventDefault();
+                onEventCapture(event);
```

Comment:
Because you're now using `onEventCapture`, you don't need the following `event.stopPropagation` statement